src/App/Tracker/Controller/Hooks/Listeners/AbstractListener.php

```diff
@@ -8,8 +8,12 @@
 
 namespace App\Tracker\Controller\Hooks\Listeners;
 
+use App\Tracker\Model\CategoryModel;
+use App\Tracker\Table\IssuesTable;
 use App\Projects\TrackerProject;
 
+use Joomla\DI\ContainerAwareInterface;
```

Comment:
i have no idea why we placed it there. But they are removed now with: https://github.com/joomla/jissues/pull/770/commits/b1d485d6db82ade99ec5ca3df7a64eacdb937fd3. Thanks.